librz/core/analysis_tp.c

```diff
@@ -564,6 +589,12 @@ void propagate_types_among_used_variables(RzCore *core, HtUP *op_cache, Sdb *tra
 	bool userfnc = false;
 	bool prop = false;
 	ut32 type = aop->type & RZ_ANALYSIS_OP_TYPE_MASK;
+
+	RzILTraceInstruction *cur_instr_trace = NULL;
+	if (ctx->cur_idx < rz_pvector_len(instr_traces)) {
+		cur_instr_trace = rz_pvector_at(instr_traces, ctx->cur_idx);
+	}
```

Comment:
Can we even continue with the rest of the function if we `cur_instr_trace` is nuLL?

Reply:
The old SDB code will return a NULL when there are no corresponding key. The function seems cover that situation. I will check them again and debug.